spec/services/user/silence_spec.rb

```diff
@@ -41,53 +42,47 @@
       it { is_expected.to fail_a_contract }
     end
 
-    context "when data is valid" do
-      context "when provided user does not exist" do
-        let(:user_id) { 0 }
+    context "when provided user does not exist" do
```

Comment:
Lately, I’ve been changing the service specs to have a structure more “flat”. For each specialized step that can fail, I create a context with a failing example and then write the next one without nesting it. I think it’s clearer, less verbose and doesn’t impact the robustness of the specs.